### DISCUSSION TITLE
Error in copy pasting, unable to go back to normal mode in Neo Vim.

### DISCUSSION THREAD

**Discussion by mayankdutta** [neovim/neovim#13985] [Q&A] [answered]
I just copied a [this code][1].cpp code from codechef and then this weird thing is going on, 
[see the pic][2] If I am trying to press any key, these ASCII symbol as shown just keep repeating. 

More specifically I am facing error while copying and pasting. 
one more issue that I was facing was during copying pasting a random character get automatically cut from file and added to the end. however this issue was less frequent, but now it is way weird. 

[Here is my init.vim][3]

  [1]: https://www.codechef.com/viewsolution/42944856
  [2]: https://i.stack.imgur.com/iBMlP.png
  [3]: https://github.com/mayankdutta/code_template/blob/master/init.vim

**Comment by dm1try** [comment, marked as answer]
@mayankdutta please provide Nvim version
might be related https://github.com/neovim/neovim/issues/11699

**Reply by dm1try** [reply]
also see https://github.com/neovim/neovim/issues/13921

**Reply by mayankdutta** [reply]
> @mayankdutta please provide Nvim version
> might be related #11699

NVIM v0.4.4
Build type: Release
LuaJIT 2.1.0-beta3
(installed via 'brew install neovim')

terminal : iterm2
operating system : macos Big sur version 11.1

issue is exactly #11699
still figuring how to fix.

Thanks for the fast reply !

**Reply by dm1try** [reply]
first, I would try a development version, see https://github.com/neovim/neovim/wiki/Installing-Neovim#macos--os-x
```
brew install --HEAD luajit
brew install --HEAD neovim
```

and follow #13921 if it will not help :)

**Reply by mayankdutta** [reply]
Thank you very much, you saved my day. 
brew install --HEAD neovim
worked for me. 
around 4 packages were broken, had to reinstall them.
